components/smugmug/actions/create-album/create-album.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "smugmug-create-album",
   name: "Create Album",
   description: "Creates an album.",
-  version: "0.4.1",
+  version: "2.0.0",
```

Comment:
Setting version to 2.0.0 since the previous published component with the same key had version 1.0.0.